common/test_rescue.yml

```diff
@@ -41,6 +41,17 @@
         download_file_fail_ignore: true
       when: vm_dir_name is defined and vm_dir_name
 
+    - include_tasks: esxi_download_datastore_file.yml
+      vars:
+        src_datastore: "{{ datastore }}"
+        src_file_path: "{{ vm_dir_name }}/{{ vm_serial_port_output_file | basename }}"
+        dest_file_path: "{{ current_test_log_folder }}/{{ vm_serial_port_output_file | basename }}"
+        download_file_fail_ignore: true
+      when: 
+        - vm_dir_name is defined
+        - vm_dir_name
+        - guest_os_family == "FreeBSD"
```

Comment:
1. vm_serial_port_output_file shall also be defined and not empty.
2. If autoinstall failed, guest_os_family will be undefined. Use `unattend_install_conf` to check it is FreeBSD here.